products/watchos.md

```diff
@@ -17,9 +17,16 @@ auto:
 -   custom: true
 
 releases:
+-   releaseCycle: "10"
+    releaseDate: 2023-09-18
+    support: true
+    eol: false
+    latestReleaseDate: 2023-09-18
+    latest: '10.0'
+
 -   releaseCycle: "9"
     releaseDate: 2022-09-12
-    support: true
+    support: 2023-09-18
     eol: false
```

Comment:
Looks like usually EOL date is set approximately one week after the end of support. Do you know the rule @BiNZGi ?

Should we set the EOL date for 9 too (and comment the release block with the support / eol date policy too) ?

Reply:
I am not sure that there is a rule about that. I also think the eol for 8 is wrong. https://en.wikipedia.org/wiki/WatchOS#Version_history lists it as still supported and latest 8.8.1 is from 2023-06-21.

@captn3m0 @hebbet What do you think?

Reply:
Let's merge for now as it was done for other apple products.

See also: https://github.com/endoflife-date/endoflife.date/pull/3581#issuecomment-1734433868.